common/chat/user.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable camelcase */
+import { GraphQLUser } from '../user/session';
+import dotenv from 'dotenv';
+import { checkResponseStatus, parseSlashToUnderscore } from './helper';
+
+dotenv.config();
+
+/**
+ * NOTE: `id` ist not the same as `userId`! It's a transformed version, e.g.
```

Comment:
It would also be good to document the 'why' ... "as TalkJS' id field must not contain slashes"